tools/tagger/tagger.js

```diff
@@ -1,55 +1,65 @@
-function sanitizeName(name) {
-  if (!name) {
-    return '';
-  }
-
-  const decodedText = decodeURIComponent(name.trim()).toLowerCase().normalize('NFD').replace(/[\u0300-\u036f]/g, '');
-  const sanitizedText = decodedText.split(',').map((text) => text.trim().replace(/[^a-z0-9]+/g, '-').replace(/^-|-$/g, '')).join(', ');
-  return sanitizedText.trim();
-}
-
 function renderItems(cat, catId, taxonomy) {
   let html = '';
-  const items = taxonomy.map((item) => item[cat]);
-  items.forEach((tag) => {
-    if (tag.trim() !== '') {
+  const oCategoryTags = taxonomy[cat];
+  oCategoryTags.forEach((tag) => {
+    if (tag.key.trim() !== '' && tag.text.trim() !== '') {
       html += `
       <span class="path">
-        <span data-title="${tag}" class="tag cat-${catId % 8}">${tag}</span>
+        <span data-title="${tag.key}" class="tag cat-${catId % 8}">${tag.text}</span>
       </span>
     `;
     }
   });
   return html;
 }
 
+const getReduceTags = (oTaxonomy, sCategory) => {
```

Comment:
any reason why prefixing variables with `o` and `s`?

Reply:
@ravkiran that means the data type of the variable, o = Object, s =String. Just a habit of mine. Please let us know if we need to update it. Thanks!